liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/AbstractLiquibaseMojo.java

```diff
@@ -839,20 +841,13 @@ public void configureFieldsAndValues() throws MojoExecutionException, MojoFailur
         if (propertyFile != null) {
             getLog().info("Parsing Liquibase Properties File");
             getLog().info("  File: " + propertyFile);
-            try (InputStream is = handlePropertyFileInputStream(propertyFile)) {
+            try (InputStream _is = handlePropertyFileInputStream(propertyFile)) {
```

Comment:
I feel I am not doing the write thing here, as `handlePropertyFileInputStream` will throw an exception if the `propertyFile` does not exist.

What do you suggest here @MalloD12

Reply:
Hi @mhewedy,

First, let me thank you for applying my previous review comments. Now, regarding this question I think what you need to do here is also catch `MojoFailureException` which will be thrown by `handlePropertyFileInputStream()` in case the file does not exists. Also, I think you can either remove `_is` or `is` variable declaration, because `_is` is defined in the try and afterwards `is` is defined and being assigned with `_is` value.

Thanks,
Daniel.

Reply:
Actually, I found that `handlePropertyFileInputStream` calls  `ResouceAccessor#getExisting` which will calls `returnEmptyResourceIfMissingFile`.

So we need no handling here.. Do you agree?

Reply:
That's correct, although I still think we should add the `MojoFailureException` to the catch block in the `configureFieldsAndValues` method. Rest of the things looks good to me. 

Thanks,
Daniel.

Reply:
Ok, will do that... 
Should I add a call to `returnEmptyResourceIfMissingFile` in the catch block?

Reply:
I meant just doing something like:  `catch (IOException  | MojoFailureException e)`. And I think doing that you can also remove `MojoFailureException` from the method signature.

Thanks,
Daniel.

Reply:
Aha, I got it...
It is updated now